frontend/src/pages/Admin/ManageCafe/components/PreviewCoupon/index.tsx

```diff
@@ -0,0 +1,24 @@
+import LoadingSpinner from '../../../../../components/LoadingSpinner';
+import { useRedirectRegisterPage } from '../../../../../hooks/useRedirectRegisterPage';
+import useGetCouponDesign from '../../hooks/useGetCouponDesign';
+import { PreviewBackImage, PreviewCouponBackImage } from './style';
+
+const PreviewCoupon = () => {
+  const cafeId = useRedirectRegisterPage();
+  const { data: couponDesignData, status: couponDesignStatus } = useGetCouponDesign(cafeId);
+
+  if (couponDesignStatus === 'loading') return <LoadingSpinner />;
+  if (couponDesignStatus === 'error') return <p>쿠폰 이미지를 불러오는 데 실패했습니다.</p>;
+
+  return (
+    <PreviewCouponBackImage>
+      {couponDesignData.backImageUrl !== '' ? (
+        <PreviewBackImage src={couponDesignData.backImageUrl} />
+      ) : (
+        <p>쿠폰 뒷면 이미지가 들어갈 공간입니다.</p>
+      )}
```

Comment:
이것도 될것 같다는 생각입니다!
```suggestion
      {!!couponDesignData.backImageUrl ? (
        <PreviewBackImage src={couponDesignData.backImageUrl} />
      ) : (
        <p>쿠폰 뒷면 이미지가 들어갈 공간입니다.</p>
      )}
```

Reply:
해당 표현은 eslint에서 redundant double negation (불필요한 이중 부정)으로 캐치하네요. `!`으로 하고 삼항연산자 내부의 순서를 바꾼다거나 하는 방법도 있겠네요.혹은 

```suggestion
      {couponDesignData.backImageUrl ? (
        <PreviewBackImage src={couponDesignData.backImageUrl} />
      ) : (
        <p>쿠폰 뒷면 이미지가 들어갈 공간입니다.</p>
      )}
```

그냥 이런 식으로 표현해도 좋을 것 같습니다

Reply:
오호!! 그렇군요~ 훨 깔끔합니다요